Splitio.Redis/Telemetry/Storages/RedisTelemetryStorage.cs

```diff
@@ -25,16 +27,27 @@ public class RedisTelemetryStorage : ITelemetryInitProducer, ITelemetryEvaluatio
             .Replace("{prefix}.", string.IsNullOrEmpty(_userPrefix) ? string.Empty : $"{_userPrefix}.");
 
         public RedisTelemetryStorage(IRedisAdapterProducer redisAdapterProducer,
-            string userPrefix,
+            RedisConfig redisConfig,
             string sdkVersion,
             string machineIp,
-            string machineName)
+            string machineName, bool clusterMode = false)
```

Comment:
you can remove `machineName`, `machineIP` and `sdkVersion` and you could use `redisConfig`